### PR TITLE
EOS-17546: Alert Sequence Bug  

### PR DESCRIPTION
**Pull request by sumedhak27**
… empty)

Signed-off-by: Sumedh A. Kulkarni <sumedh.a.kulkarni@seagate.com>

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ * ] Yes
    - [ ] No
  </code>
</pre>
### Message Seqence Before Change : 
 [root@ssc-vm-2215 cortx-monitor]# low-level/tests/manual/rabbitmq_reader.py sspl-out sensor-key
^[[A
{"title": "SSPL Sensor Response", "description": "Seagate Storage Platform Library - Sensor Response", "username": "sspl-ll", "signature": "None", "time": "1613124954", "expires": 3600, "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"service_watchdog": {"service_name": "rabbitmq-server.service", "service_state": "deactivating", "previous_service_state": "deactivating", "service_substate": "stop-sigterm", "previous_service_substate": "stop", "pid": "21552", "previous_pid": "21552"}}}}
{"title": "SSPL Sensor Response", "description": "Seagate Storage Platform Library - Sensor Response", "username": "sspl-ll", "signature": "None", "time": "1613124956", "expires": 3600, "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"service_watchdog": {"service_name": "rabbitmq-server.service", "service_state": "activating", "previous_service_state": "deactivating", "service_substate": "start", "previous_service_substate": "stop-sigterm", "pid": "21552", "previous_pid": "21552"}}}}
{"title": "SSPL Sensor Response", "description": "Seagate Storage Platform Library - Sensor Response", "username": "sspl-ll", "signature": "None", "time": "1613124958", "expires": 3600, "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"service_watchdog": {"service_name": "rabbitmq-server.service", "service_state": "active", "previous_service_state": "activating", "service_substate": "running", "previous_service_substate": "start", "pid": "21552", "previous_pid": "21552"}}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124958", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124952", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 14"**`, "IEC": "EO0090090900"}, "alert_id": "161312495881bd758fe3874ddea174a2ea7cb8372d", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124958", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124952", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 15"**`, "IEC": "EO0090090900"}, "alert_id": "161312495861178e8e00824cd78f948e314e457f32", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124958", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124952", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 16"**`, "IEC": "EO0090090900"}, "alert_id": "1613124958cbefb2474d9c4324acb23b9f95fd9704", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124959", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124952", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 17"**`, "IEC": "EO0090090900"}, "alert_id": "1613124958772d6b71fd5e41cc9219562c7ce78974", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124947", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 1"**`, "IEC": "EO0090090900"}, "alert_id": "16131249477b9fe95e5af449a5bc2b0cdf6219995b", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124947", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 2"**`, "IEC": "EO0090090900"}, "alert_id": "1613124947983f5394e9264898ac3b7eed07c576d4", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124947", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 3"**`, "IEC": "EO0090090900"}, "alert_id": "1613124947ea5c47fb909840e3b9fca8c74f3ea7be", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124947", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 4"**`, "IEC": "EO0090090900"}, "alert_id": "16131249474b8038d1838c4640b34a23a94003dfc9", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124947", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 5"**`, "IEC": "EO0090090900"}, "alert_id": "1613124947cdf02681f412458b8000bced6538c133", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 6"**`, "IEC": "EO0090090900"}, "alert_id": "1613124947e7b9d9e2c15544329040497125c1b72a", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 7"**`, "IEC": "EO0090090900"}, "alert_id": "16131249470bd1fe182ebe43c0b2ba333b72afbbea", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 8"**`, "IEC": "EO0090090900"}, "alert_id": "161312494709ed6ab7c81c4402a8f027ce08b1f10d", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 9"**`, "IEC": "EO0090090900"}, "alert_id": "16131249473d8465cc9dd64ae198529bb678b0aef5", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 10"**`, "IEC": "EO0090090900"}, "alert_id": "1613124948fee42099cc4d4e0689327b0ead8c385a", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 11"**`, "IEC": "EO0090090900"}, "alert_id": "16131249480d56a7a03be34c89b6dedf6640410fd6", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 12"**`, "IEC": "EO0090090900"}, "alert_id": "1613124948554001f64044494a8183c053286c0cce", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613124948", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613124947", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 13"**`, "IEC": "EO0090090900"}, "alert_id": "1613124948605be3d98a024241a2dbcaf90a786b88", "host_id": "ssc-vm-2215.colo.seagate.com"}}}

### Message sequence after the change
[root@ssc-vm-2215 cortx-monitor]# low-level/tests/manual/rabbitmq_reader.py sspl-out sensor-key
^[[A
{"title": "SSPL Sensor Response", "description": "Seagate Storage Platform Library - Sensor Response", "username": "sspl-ll", "signature": "None", "time": "1613125682", "expires": 3600, "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"host_id": "ssc-vm-2215.colo.seagate.com", "alert_type": "fault", "alert_id": "1613125681bb23bae899e7427a95f5b9f96071a746", "severity": "critical", "info": {"site_id": "DC01", "rack_id": "RC01", "node_id": "SN01", "cluster_id": "CC01", "resource_id": "0", "resource_type": "enclosure", "event_time": "1613125681"}, "specific_info": {"event": "Storage Enclosure unreachable,Possible causes : Enclosure / Storage Controller /Management Controller rebooting,Network port blocked by firewall,Network outage or Power outage."}}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 1"**`, "IEC": "EO0090090900"}, "alert_id": "16131256820d381935b50349d59669e3132decccaf", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 2"**`, "IEC": "EO0090090900"}, "alert_id": "161312568262c86521fe5e4647b241e64fb483c604", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 3"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682d694d3868d8e4822b704a09d935c312d", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 4"**`, "IEC": "EO0090090900"}, "alert_id": "16131256821e970002e64941bc9243b084dff79c57", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 5"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682d223caaf17514379bca27a5523816553", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 6"**`, "IEC": "EO0090090900"}, "alert_id": "16131256829c2f8822e6e24412ba36e3aed52f8c36", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 7"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682ffcc5b6e8cc94eecb2a0e4a804466cbf", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 8"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682391acb42668540b38f037914f29d1e72", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 9"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682a1c3a251dc6142bbaa828355b7b3a8f6", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 10"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682645e4aa045114a72b82be43d198e9add", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 11"**`, "IEC": "EO0090090900"}, "alert_id": "1613125682a2e809b08658418e9f7f5936721d8768", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 12"**`, "IEC": "EO0090090900"}, "alert_id": "16131256829d5ac463b4594fb890adeea7bfaf1d99", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125682", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125680", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 13"**`, "IEC": "EO0090090900"}, "alert_id": "16131256827d94dbe8c0e74482bd46d90f948dedf2", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125693", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125684", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 14"**`, "IEC": "EO0090090900"}, "alert_id": "1613125692582b8c68391c4d5983345b9050e0c954", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125693", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125684", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 15"**`, "IEC": "EO0090090900"}, "alert_id": "16131256927c7161cdfe8c4cbbac48d5ae42765339", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125693", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125684", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900", `"description": **"Test 16"**`, "IEC": "EO0090090900"}, "alert_id": "16131256928bf1ea52a70e462594a6d299dc98673e", "host_id": "ssc-vm-2215.colo.seagate.com"}}}
{"username": "sspl-ll", "description": "Seagate Storage Platform Library - Sensor Response", "title": "SSPL Sensor Response", "expires": 3600, "signature": "None", "time": "1613125693", "message": {"sspl_ll_msg_header": {"schema_version": "1.0.0", "sspl_version": "1.0.0", "msg_version": "1.0.0"}, "sensor_response_type": {"info": {"event_time": "1613125684", "resource_id": "iem", "site_id": "DC01", "node_id": "SN01", "cluster_id": "CC01", "rack_id": "RC01", "resource_type": "iem"}, "alert_type": "fault", "severity": "error", "specific_info": {"source": "OS", "component": "009", "module": "009", "event": "0900",` "description": **"Test 17"**`, "IEC": "EO0090090900"}, "alert_id": "1613125692af346f1866ef44ed89399a42df828388", "host_id": "ssc-vm-2215.colo.seagate.com"}}}

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: low-level/framework/rabbitmq/rabbitmq_egress_processor.py [hunk 1]
```diff
@@ -276,10 +276,13 @@ def _transmit_msg_on_exchange(self):
...
-                    self._connection.publish(exchange=self._exchange_name,
-                                            routing_key=self._routing_key,
-                                            properties=msg_props,
-                                            body=jsonMsg)
+                    if self.store_queue.is_empty():
+                        self._connection.publish(exchange=self._exchange_name,
+                                                routing_key=self._routing_key,
+                                                properties=msg_props,
+                                                body=jsonMsg)
+                    else:
```
Comment (cdeshmukh) [inline low-level/framework/rabbitmq/rabbitmq_egress_processor.py:284]: good. add a log.info() also here stating the action, with reason. keep it short

File: low-level/framework/rabbitmq/rabbitmq_egress_processor.py [hunk 2]
```diff
@@ -276,10 +276,13 @@ def _transmit_msg_on_exchange(self):
...
+                        self.store_queue.put(jsonMsg)
```
[the remark this answers is not in the document]
Comment (cdeshmukh) [reply low-level/framework/rabbitmq/rabbitmq_egress_processor.py:285]: that would be additional hop for regular cases

File: low-level/framework/rabbitmq/rabbitmq_egress_processor.py (hunk 1 above)
Comment (sumedhak27) [reply low-level/framework/rabbitmq/rabbitmq_egress_processor.py:284]: added.

**Review by cdeshmukh — APPROVED** [review]

**Review by sandeepsgt — APPROVED** [review]

**Pull request merged**